static/workspace-view.less

```diff
@@ -1,4 +1,19 @@
 @import "ui-variables";
+@import "octicon-mixins";
```

Comment:
Do you mind removing all these additions in this file?

Sorry, there has been some major reshuffling of the core styles and all these styles are now in [scaffolding.less](https://github.com/atom/atom/blob/master/static/scaffolding.less)

 It's probably also part of the:

> seems like a bad rebase conflict resolution.